discord/abc.py

```diff
@@ -1401,6 +1407,8 @@ async def send(
             A list of stickers to upload. Must be a maximum of 3.
 
             .. versionadded:: 2.0
+        suppress: :class:`bool`
+            Whether to suppress embeds for the message.
```

Comment:
```suggestion
            Whether or not embeds should be suppressed for this message.
```

Reply:
I suggested that line to make this consistent with `Message.edit`, `PartialMessage.edit` and `Paginator.edit`